tests/test_telemetry.py

```diff
@@ -1,12 +1,11 @@
 from utils import context, BaseTestCase, interfaces, missing_feature, bug, released, flaky, irrelevant
 
 
-@released(dotnet="2.12.0", java="0.108.1")
+@released(dotnet="2.12.0", java="0.108.1", nodejs="3.2.0")
```

Comment:
@cbeauchesne Can we add a `@bug` here that checks something like "if scenario == UDS && if library version < xxx"

Reply:
```
@bug(context.scenario == "UDS" and context.library < "nodejs.x.y.z")
```

😎

Reply:
Update this when we know the actual released version:
```suggestion
@released(dotnet="2.12.0", java="0.108.1", nodejs="3.2.0")
@bug(context.scenario == "UDS" and context.library < "nodejs@3.7.0")
```